doc/source/plugins/b610_django_extra_used.rst

```diff
@@ -0,0 +1,5 @@
+-----------------------
+B610: django_extra_used
+-----------------------
+
+.. automodule:: bandit.plugins.django_injection_sql
```

Comment:
Because you have two functions in this module, you should instead use:

.. currentmodule:: bandit.plugins.django_injection_sql

.. autofunction:: django_extra_used
   :noindex: